R/match.R

```diff
@@ -0,0 +1,101 @@
+#' Test for presence of certain groups/levels in a vector
+#'
+#' A validated version of `%in%` intended for use with logical subsetting.
+#' Performs matching between a factor and a set of levels to pose the question
+#' "which of the elements of a factor match these levels?"
+#'
+#' @md
+#' @rdname fct_match
+#' @param f A factor (or character vector).
+#' @param inverse (logical) if `TRUE`, perform a "not match" operation.
+#' @param allow_missing (logical) if `TRUE`, allow matching to empty levels in
+#'   f
+#' @param ... A series of characters for which the test "are these levels in
```

Comment:
Are you sure this should be `...` and not a single vector? Using `...` saves typing but it makes it feel a bit weird to me.